js_modules/dagster-ui/packages/ui-core/src/nav/LatestRunTag.tsx

```diff
@@ -117,10 +117,10 @@ export const LatestRunTag = ({
   );
 };
 
-const LATEST_RUN_TAG_QUERY = gql`
+export const LATEST_RUN_TAG_QUERY = gql`
   query LatestRunTagQuery($runsFilter: RunsFilter) {
     pipelineRunsOrError(filter: $runsFilter, limit: 1) {
-      ... on PipelineRuns {
+      ... on Runs {
```

Comment:
This fragment broke mocking, because `PipelineRuns` is an interface and not a concrete type. Fixed by changing the fragment, and I believe it should all be fine downstream.